R/git.R

```diff
@@ -532,7 +532,7 @@ git_sitrep <- function() {
   ## TODO: rework when ui_*() functions make it possible to do better
   branch <- if (is.null(branch)) "<unset>" else branch
   tracking_branch <- if (is.null(tracking_branch)) "<unset>" else tracking_branch
-  cat_line("* ", "Local branch -> remote tracking branch: ",
+  ui_inform("* ", "Local branch -> remote tracking branch: ",
```

Comment:
Just an observation, with the caveat that I haven't recently re-read the `ui_*()` functions. When I see these hand-crafted bullets, I wonder if we should be using `ui_bullet()`?

Oh maybe it's not the same type of bullet. Maybe we need something like `ui_item()` for these sorts of bullet lists. We do this a lot.

Reply:
Is that just `ui_info()`?

Reply:
Yeah, it seems like sitrep could do with a UI pass a some point.

Reply:
I think we also just gained hand-crafted bullets in the PR about merge conflicts. There are probably many instances.